### PR TITLE
Surchage classes and add constants

### PR DESCRIPTION
**Pull request by S0dlive**
J'ai aussi corrigée la class Command .

### CODE REVIEW

File: Bird/Command.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 namespace Bird
 {
-    public class Command
+    public abstract class Command
```
Comment (Kiirx) [inline Bird/Command.cs:5]: Why should it be abstract if we dont use it?

File: Bird/Bird.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@ namespace Bird
 {
     public class Bird
     {
-        public readonly string version = "v1.0.1";
+        public const string version = "v1.0.1"; // Il y a pas de ctor  donc . . .
```
Comment (Kiirx) [inline Bird/Bird.cs:10]: Please set this comment in **ENGLISH**

File: Bird/Bird.cs [hunk 3]
```diff
@@ -164,16 +180,25 @@ public List<string> ParseCommandLine(string cmdLine)
             Console.BackgroundColor = ConsoleColor.Black;
         }
 
-        public void WriteLineChar(char character, ConsoleColor foregroundColor = ConsoleColor.White,
+        /// <summary>
+        ///     Output text with color
+        /// </summary>
+        /// <param name="character">The char to output</param>
```
Comment (Kiirx) [inline Bird/Bird.cs:186]: This parameter needs to be changed to number.

File: Bird/Bird.cs [hunk 4]
```diff
@@ -164,16 +180,25 @@ public List<string> ParseCommandLine(string cmdLine)
...
+        /// <param name="foregroundColor">Change foreground text color</param>
+        /// <param name="backgroundColor">Change background text color</param>
+        public void WriteLine(int number, ConsoleColor foregroundColor = ConsoleColor.White,
             ConsoleColor backgroundColor = ConsoleColor.Black)
         {
             Console.ForegroundColor = foregroundColor;
             Console.BackgroundColor = backgroundColor;
-            Console.Write(character + "\n");
+            Console.Write(number);
```
Comment (Kiirx) [inline Bird/Bird.cs:194]: This isnt good, its a WriteLine method, and so, a \n is required.

**Review by Kiirx — CHANGES_REQUESTED** [review]
Command changes are to clarify, WriteLine(int number) needs to be changes

File: Bird/Command.cs (hunk 1 above)
Comment (S0dlive) [reply Bird/Command.cs:5]: simplement parce que je ne vois pas l'intérêt d'avoir une class instanciable quand derrière tu fais du polymorphisme (x

Comment (Kiirx) [reply Bird/Command.cs:5]: that's nonsense, I still dont see why you make it abstract

Comment (S0dlive) [reply Bird/Command.cs:5]: so you created a class that aims to do polymorphism and you make it instantiable? Also, your class doesn't have a ctor. . .

**Comment by Kiirx** [comment]
I can merge this if you fix your code.

**Pull request closed without merging**